modules/install-dnsmasq/install-dnsmasq

```diff
@@ -86,7 +86,7 @@ function install_dnsmasq {
     sudo yum update -y
     sudo yum install -y dnsmasq
     echo "prepend domain-name-servers $consul_ip;" | sudo tee -a "/etc/dhcp/dhclient.conf" > /dev/null
-    echo "conf-dir=/etc/dnsmasq.d" | sudo tee -a "/etc/dnsmasq.conf" > /dev/null
+    echo "conf-dir=$DNS_MASQ_CONFIG_DIR" | sudo tee -a "/etc/dnsmasq.conf" > /dev/null
```

Comment:
Instead of an somewhat hidden environment variable that no one will know to set, you could you add this as a command-line arg to the script (following the pattern of all the args already there) and pass it through to this function?

Reply:
This variable is already set on [line 14](https://github.com/aaron0browne/terraform-aws-consul/blob/0a5e90055b8c38d18a5543d73fefcaf11347752a/modules/install-dnsmasq/install-dnsmasq#L14).

Reply:
Ohh, haha, you're right. OK, looks like I missed that while trying to make the code more DRY :)